docs/modules/usage/prompting/repo-specific-customization.md

```diff
@@ -0,0 +1,140 @@
+# Repository-Specific Prompt Customization
+
+OpenHands provides two powerful ways to customize its behavior for specific repositories:
+
+1. `.openhands_instructions` file for repository-wide guidelines
+2. `.openhands` directory for custom micro-agents
+
+## The `.openhands_instructions` File
+
+The `.openhands_instructions` file is a simple text file placed in the root directory of your repository that provides repository-specific context and guidelines to OpenHands. This file's contents are automatically injected into the prompt when OpenHands processes issues or pull requests.
+
+### What to Include
+
+Your `.openhands_instructions` file should contain:
```

Comment:
```suggestion
Your `.openhands_instructions` file can contain information like the following:
```

Reply:
Openhands fix success summary


The feedback appears to be suggesting a specific formatting change to introduce code block syntax (using triple backticks) for the `.openhands_instructions` file example. However, the AI's response doesn't directly address this formatting suggestion - instead it talks about updating the content of the example and simplifying other sections.

While the AI may have made other improvements to the documentation, it hasn't specifically implemented the requested formatting change to wrap the `.openhands_instructions` file content in code blocks. The feedback was very specific about syntax (```suggestion) which indicates this was meant to be a formatting change, not a content change.

To properly resolve this feedback, the AI should add the triple backtick syntax around the `.openhands_instructions` file example in the documentation.